packages/backend/src/api/controllers/report/generateTvlResponse.ts

```diff
@@ -1,37 +1,40 @@
 import {
-  ApiMain,
-  MainChart,
-  MainChartPoint,
-  MainCharts,
   ProjectId,
+  TvlApiChart,
+  TvlApiChartPoint,
+  TvlApiCharts,
+  TvlApiResponse,
 } from '@l2beat/types'
 
 import { AggregateReportRecord } from '../../../peripherals/database/AggregateReportRepository'
 import { ReportRecord } from '../../../peripherals/database/ReportRepository'
 import { asNumber } from './asNumber'
 import { getChartPoints } from './charts'
 
-export function generateMain(
+export function generateTvlResponse(
```

Comment:
`generateTvlApiResponse`. This applies to many more things. I believe that we shouldn't make an exception for variable names and include the full type name everywhere. It might be three letters longer but it avoids confusion as to which form should be used.

Reply:
I like it 👍